hudi-common/src/main/java/org/apache/hudi/common/util/JsonUtils.java

```diff
@@ -35,6 +36,7 @@ public class JsonUtils {
   private static final ObjectMapper MAPPER = new ObjectMapper();
 
   static {
+    MAPPER.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
```

Comment:
this feature is default true even in old version (2.6) but does not seem to be effective. so it's effectively false. needs to add more test to verify the impact